### PR TITLE
Fixed damage to players via fire and tnt entities. Maybe fixed vault.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Update: Fixed all bugs.

File: src/main/java/com/gmail/tracebachi/FloorIsLava/Arena.java [hunk 1]
```diff
@@ -109,16 +111,16 @@ public Arena(FloorIsLavaPlugin plugin)
...
-        if(amount == null || amount <= 0)
+        if (amount == null || amount <= 0)
         {
             player.sendMessage(BAD + "That is not a valid amount to wager.");
             return;
         }
 
         String name = player.getName();
-        EconomyResponse response = plugin.getEconomy().bankWithdraw(name, amount);
+        EconomyResponse response = plugin.getEconomy().withdrawPlayer(Bukkit.getOfflinePlayer(name), amount);
```
Comment (GeeItsZee) [inline src/main/java/com/gmail/tracebachi/FloorIsLava/Arena.java]: @Fridgey There is a name version of `withdrawPlayer()`. Use that one.

File: src/main/java/com/gmail/tracebachi/FloorIsLava/Arena.java [hunk 2]
```diff
@@ -109,16 +111,16 @@ public Arena(FloorIsLavaPlugin plugin)
 
     public void addWager(Integer amount, Player player)
     {
-        if(amount == null || amount <= 0)
+        if (amount == null || amount <= 0)
```
Comment (GeeItsZee) [inline src/main/java/com/gmail/tracebachi/FloorIsLava/Arena.java]: @Fridgey Can you fix the formatting so there is no space after `if` and `for` and etc?

File: src/main/java/com/gmail/tracebachi/FloorIsLava/Arena.java [hunk 3]
```diff
@@ -307,13 +311,7 @@ public void tick()
...
                 Firework firework = player.getWorld().spawn(player.getLocation().add(0, 1, 0), Firework.class);
                 FireworkMeta fireworkMeta = firework.getFireworkMeta();
-                fireworkMeta.addEffects(FireworkEffect.builder()
-                    .flicker(false)
-                    .trail(true)
-                    .with(Type.BALL_LARGE)
-                    .withColor(Color.BLUE)
-                    .withFade(Color.WHITE)
-                    .build());
+                fireworkMeta.addEffects(FireworkEffect.builder().flicker(false).trail(true).with(Type.BALL_LARGE).withColor(Color.BLUE).withFade(Color.WHITE).build());
```
Comment (GeeItsZee) [inline src/main/java/com/gmail/tracebachi/FloorIsLava/Arena.java]: @Fridgey Avoid long lines like this. Use the format I had before. It's easier to read.

File: src/main/java/com/gmail/tracebachi/FloorIsLava/Arena.java [hunk 4]
```diff
@@ -392,27 +390,25 @@ public void loadConfig(FileConfiguration config)
         invisUseDelay = config.getInt("RodOfInvisibilityUseDelay");
         boostUseDelay = config.getInt("BoostUseDelay");
 
-        arenaBlocks = new ArenaBlocks(
-            config.getConfigurationSection("PointOne"),
-            config.getConfigurationSection("PointTwo"));
+        arenaBlocks = new ArenaBlocks(config.getConfigurationSection("PointOne"), config.getConfigurationSection("PointTwo"));
```
Comment (GeeItsZee) [inline src/main/java/com/gmail/tracebachi/FloorIsLava/Arena.java]: @Fridgey Long line.

File: src/main/java/com/gmail/tracebachi/FloorIsLava/Arena.java [hunk 5]
```diff
@@ -476,33 +472,33 @@ public void onPlayerBlockClick(PlayerInteractEvent event)
...
                 event.getClickedBlock().setType(Material.AIR);
             }
 
             return;
         }
 
-        if(heldItem == null ||
-            (event.getAction() != Action.RIGHT_CLICK_AIR &&
-            event.getAction() != Action.RIGHT_CLICK_BLOCK)) return;
+        if (heldItem == null || (event.getAction() != Action.RIGHT_CLICK_AIR && event.getAction() != Action.RIGHT_CLICK_BLOCK))
```
Comment (GeeItsZee) [inline src/main/java/com/gmail/tracebachi/FloorIsLava/Arena.java]: @Fridgey Long line.

**Comment by ghost** [comment]
Should be all better now if i didn't miss anything. Don't know what happened with all of the formatting though, must have been automatically done in my IDE.

**Comment by GeeItsZee** [comment]
LGTM :+1: I'll load this in later today.

**Pull request merged**